libsolidity/ast/ASTJsonConverter.h

```diff
@@ -155,6 +158,12 @@ class ASTJsonConverter: public ASTConstVisitor
 		std::vector<std::pair<std::string, Json::Value>> &_attributes,
 		ExpressionAnnotation const& _annotation
 	);
+	static void appendMove(Json::Value& _array, Json::Value&& _value)
+	{
+		solAssert(_array.isArray(), "");
```

Comment:
This is explicit now and hence we need to create the "children" arrays above, however if we remove the assert here, jsoncpp will internally just created it on `append`. Not sure which is faster, but this is more explicit that's for sure.